components/studio/plan.sh

```diff
@@ -9,33 +10,36 @@ pkg_build_deps=(core/coreutils core/tar core/xz core/wget core/busybox-static co
 pkg_bin_dirs=(bin)
 
 do_build() {
-  cp -v $PLAN_CONTEXT/bin/hab-studio.sh hab-studio
-  cp -v $PLAN_CONTEXT/libexec/hab-studio-profile.sh .
-  cp -v $PLAN_CONTEXT/libexec/hab-studio-type-*.sh .
+  cp -v "$PLAN_CONTEXT"/bin/hab-studio.sh hab-studio
+  cp -v "$PLAN_CONTEXT"/libexec/hab-studio-profile.sh .
+  cp -v "$PLAN_CONTEXT"/libexec/hab-studio-type-*.sh .
 
   # Embed the release version and author information of the program.
+  # shellcheck disable=2154
   sed \
     -e "s,@author@,$pkg_maintainer,g" \
     -e "s,@version@,$pkg_version/$pkg_release,g" \
     -i hab-studio
 }
 
 do_install() {
-  install -v -D hab-studio $pkg_prefix/bin/hab-studio
-  install -v -D hab-studio-profile.sh $pkg_prefix/libexec/hab-studio-profile.sh
-  for f in `ls hab-studio-type-*.sh`; do
-    install -v -D $f $pkg_prefix/libexec/$f
+  # shellcheck disable=2154
+  install -v -D hab-studio "$pkg_prefix"/bin/hab-studio
+  install -v -D hab-studio-profile.sh "$pkg_prefix"/libexec/hab-studio-profile.sh
+  for f in hab-studio-type-*.sh; do
```

Comment:
See https://github.com/koalaman/shellcheck/wiki/SC2045#iterating-over-ls-output-is-fragile-use-globs